keyboards/waldo/keymaps/default_split_shft_bck/config.h

```diff
@@ -0,0 +1,6 @@
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "config_common.h"
```

Comment:
`keyboards/waldo/config.h` includes this file automatically, so it's redundant here.

```suggestion
```